client/components/plans/plan-actions/index.jsx

```diff
@@ -159,35 +157,6 @@ var PlanActions = React.createClass( {
 		page( '/checkout/' + this.props.site.slug );
 	},
 
-	handleClickStartTrial: function( event ) {
-		event.preventDefault();
-
-		if ( this.props.isSubmitting ) {
-			return;
-		}
-
-		this.recordStartFreeTrialClick();
-
-		if ( this.props.onSelectPlan ) {
-			return this.props.onSelectPlan( this.getCartItem( { isFreeTrial: true } ) );
-		}
-
-		upgradesNotices.displaySubmitting( { isFreeCart: true } );
-
-		upgradesActions.startFreeTrial( this.props.site.ID, this.props.plan, function( error ) {
-			if ( error ) {
-				upgradesNotices.displayError( error );
-				return;
-			}
-
-			upgradesNotices.clear();
-
-			this.props.refreshSitePlans( this.props.site.ID );
```

Comment:
The `refreshSitePlans` prop should be removed as well. It means we can get rid of the Redux `connect()` part.